content/hardware/03.nano/boards/nano/tech-specs.yml

```diff
@@ -21,6 +21,6 @@ Clock speed:
 Memory:
   ATmega328P: 2KB SRAM, 32KB flash 1KB EEPROM
 Dimensions:
-  Weight: 5gr
+  Weight: 7 gr
```

Comment:
Actually, shall we use `g` instead of `gr`?

Reply:
That makes probably more sense :+1: